io/io/src/RRawFileUnix.cxx

```diff
@@ -88,6 +92,21 @@ void ROOT::Internal::RRawFileUnix::OpenImpl()
    }
 }
 
+void ROOT::Internal::RRawFileUnix::ReadVImpl(RIOVec *ioVec, unsigned int nReq)
+{
+#ifdef R__HAS_URING
+   if (!RIoUring::IsAvailable()) {
+      RRawFile::ReadVImpl(ioVec, nReq);
```

Comment:
Let's add an `R__DEBUG_HERE` statement to let us know that we cannot use io_uring